sections/futures/MarketDetails/MarketDetails.tsx

```diff
@@ -146,12 +150,12 @@ const MarketDetails: React.FC<MarketDetailsProps> = ({ baseCurrencyKey }) => {
 				),
 			},
 			'Funding Rate': {
```

Comment:
I think this should be `'Funding Rate 24h'`, because 24h is not the usual default value (and there is no agreed time unit for this AFAIK). Maybe even `Funding 24h` would be acceptable if short version needed.

A detailed tooltip is still in order I think, but adding the time unit is the very minimum IMO.